OptimizelySDK/OptimizelyDecisions/DefaultDecisionReasons.cs

```diff
@@ -0,0 +1,63 @@
+﻿/* 
+ * Copyright 2020, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+
+namespace OptimizelySDK.OptimizelyDecisions
+{
+    public class DefaultDecisionReasons : IDecisionReasons
+    {
+        private List<string> Errors = new List<string>();
+        private List<string> Logs = new List<string>();
```

Comment:
Can we change this to "infos" ("logs" may be confusing with log messages). I'll change the same for java-sdk as well.